vanilla_first_setup/vanilla-first-setup.gresource.xml

```diff
@@ -29,6 +29,7 @@
   </gresource>
   <gresource prefix="/org/vanillaos/FirstSetup/icons/symbolic/apps/">
     <file preprocess="xml-stripblanks" alias="org.gnome.Software-symbolic.svg">assets/symbolic/apps/org.gnome.Software-symbolic.svg</file>
+    <file preprocess="xml-stripblanks" alias="org.gnome.Decibels.png">assets/bundle-app-icons/org.gnome.Decibels.svg</file>
```

Comment:
Please move this line to the other bundled app icons.
Also preprocess="xml-stripblanks" only makes sense for .svg files so please remove that.